src/groups/mod.rs

```diff
@@ -527,27 +517,27 @@ fn twist() -> Fq2 {
 #[inline]
 fn two_inv() -> Fq {
     const_fq([
-        9781510331150239090,
-        15059239858463337189,
-        10331104244869713732,
-        2249375503248834476,
+        0x9e10460b6c3e7ea4,
+        0xcbc0b548b438e546,
+        0xdc2822db40c0ac2e,
+        0x183227397098d014,
     ])
 }
 
 #[inline]
 fn twist_mul_by_q_x() -> Fq2 {
```

Comment:
hey @m-kus great job :+1: what are those two values: `twist_mul_by_q_x` and `twist_mul_by_q_y`? I also tried to patch this lib by replacing the montgomery form but couldn't find what are those. Thanks

Reply:
Hey! That's a good question, tbh I just converted everything from Montgomery form to the plain one. I guess these coefficients are related to the sextic twist, i.e. transforming coordinates from Fq12 to Fq2